huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_opengauss_instance.go

```diff
@@ -371,6 +376,7 @@ func buildOpenGaussInstanceCreateOpts(ctx context.Context, d *schema.ResourceDat
 		ConfigurationId:     d.Get("configuration_id").(string),
 		ShardingNum:         d.Get("sharding_num").(int),
 		CoordinatorNum:      d.Get("coordinator_num").(int),
+		ReplicaNum:          d.Get("replica_num").(int), // In prePaid payment, the replica number is required.
```

Comment:
looks this parameter won't be empty, so this comment is unnecessary

Reply:
I just wanted to explain why this parameter needs to be supported even the default value of replica number is fixed to three in API and schema.